File: core/src/si/feri/slidegame/common/GameManager.java
package si.feri.slidegame.common;

import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.Preferences;
import si.feri.slidegame.MyGdxGame;

public class GameManager {
    public static final GameManager INSTANCE = new GameManager();

    //
    //
    //

    private final Preferences PREFS;

    //
    //
    //

    private GameManager() {
        PREFS = Gdx.app.getPreferences(MyGdxGame.class.getSimpleName());
    }
}

File: core/src/si/feri/rrizemljevid/common/GameManager.kt
package si.feri.rrizemljevid.common

import com.badlogic.gdx.Gdx
import com.badlogic.gdx.Preferences
import si.feri.rrizemljevid.MyGdxGame

class GameManager private constructor() {
    //
    //
    //
    private val PREFS: Preferences

    //
    //
    //
    init {
        PREFS = Gdx.app.getPreferences(MyGdxGame::class.java.simpleName)
    }

    companion object {
        val INSTANCE = GameManager()
    }
}